bskyweb/templates/post.html

```diff
@@ -56,6 +56,41 @@
   <meta property="article:published_time" content="{{ postView.IndexedAt }}">
   <link rel="alternate" type="application/json+oembed" href="https://embed.bsky.app/oembed?format=json&url={{ postView.Uri | urlencode }}" />
   <link rel="alternate" href="{{ postView.Uri }}" />
+  <script type="application/ld+json">
+    {
+      "@context": "https://schema.org",
+      "@type": "DiscussionForumPosting",
+      "author": {
+        "@type": "Person",
+        "name": "{{ postView.Author.DisplayName }}",
+        "url": "https://bsky.app/profile/{{ postView.Author.Handle }}"
+      },
+      {%- if postText %}
+      "text": "{{ postText }}",
+      {% endif %}
+      {%- if imageThumbUrls %}
+      "image": "{{ imageThumbUrls[0] }}",
```

Comment:
Thought: Does `imageThumbUrls` ever return something besides an array? I'm sure this is safe but something worth sanity checking